resources/qml/RoomDirectory.qml

```diff
@@ -0,0 +1,63 @@
+// SPDX-FileCopyrightText: 2021 Nheko Contributors
+//
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+import QtQuick 2.9
+import QtQuick.Controls 2.3
+import im.nheko 1.0
+
+Popup {
```

Comment:
Imo this should be an applicationWindow instead, so that you can join multiple rooms easily :3